code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -165,8 +165,36 @@
 			scrub(AT.return_air())
 
 /obj/machinery/portable_atmospherics/scrubber/huge/attackby(obj/item/W, mob/user)
-	if(default_unfasten_wrench(user, W))
-		if(!movable)
-			on = FALSE
+	if((W.tool_behaviour == TOOL_WRENCH) && !(machine_stat & BROKEN))
```

Comment:
flip the logic so you're early returning here and not needing the `else return` at the bottome.